src/events/types.ts

```diff
@@ -0,0 +1,108 @@
+export type CollectionEventBusPayload = {
+  eventType: EventBridgeEventType;
+  object_version: string;
+  collection: Collection;
+};
+
+export enum EventBridgeEventType {
+  COLLECTION_CREATED = 'collection-created',
+  COLLECTION_UPDATED = 'collection-updated',
+}
+
+/**
+ * This Collection type aligns with what we have in collection-api i.e the object keys are the same as the Collection type.
+ * The child types however map to the snowplow schema for the Collection object. These types should be refactored @Herraj
+ */
+export type Collection = {
+  externalId: string;
```

Comment:
should this be mapped to `collection_id` as in snowlow schema?

Reply:
yes it should and it's being done in a separate function which generates the payload for eventbridge!